kong.conf.default

```diff
@@ -385,6 +385,13 @@
                                  # to be sent across from CP to DP in Hybrid mode
                                  # Default is 4Mb - 4 * 1024 * 1024 due to historical reasons
 
+#cluster_dp_labels =             # Labels for the data plane. These key-value entries are
+                                 # exposed via the clustering/data_planes endpoint.
+                                 # The value must be configured as a string in JSON format.
+                                 #
+                                 # Example:
+                                 # `{ "deployment" : "aws", "region" : "us-east-1" }`
```

Comment:
- This works only with hybrid mode deployments with Konnect.
- Not exposed on clustering/data_planes
- values follow AIP standards - link to the appropriate web page